tsktsk/commands/base.py

```diff
@@ -204,14 +234,15 @@ def edit_dependencies(task: Task, add: Set[str], remove: Set[str]):
 def list() -> None:
     "List tasks to be done, with highest value:effort ratio first."
 
-    sorted_tasks = sort_tasks_by_roi(tasks())
+    repo = tasks()
+    sorted_tasks = sort_tasks_by_roi(repo)
 
     if not sorted_tasks:
         click.echo("No tasks", err=True)
         return
 
-    for task in sorted_tasks:
-        click.echo(task)
+    for task, eta in sequential_eta(repo, sorted_tasks):
```

Comment:
How would things work if velocity was an item like `repo` ?

It would mean this method would change to something like
```python
repo = tasks()
velos = velocities(repo)

for task in sorted_tasks:
  click.echo(describe-task(task, velos.eta(task)))
```

How would it affect other parts of this diff?